packages/ui/package.json

```diff
@@ -25,6 +27,7 @@
     "@logto/phrases": "workspace:^",
     "@logto/phrases-ui": "workspace:^",
     "@logto/schemas": "workspace:^",
+    "@logto/shared": "workspace:^",
```

Comment:
Seems no reference to @logto/shared under the UI package.

Reply:
yes, seems i removed the reference somehow. actually we can have a lint rule about extra dependencies. but no hurries

Reply:
maybe https://github.com/depcheck/depcheck